src/Bathymetry.jl

```diff
@@ -93,17 +93,10 @@ function regrid_bathymetry(target_grid;
                            major_basins = Inf) # Allow an `Inf` number of ``lakes''
 
     filepath = joinpath(dir, filename)
-    fileurl  = joinpath(url, filename)
-
-    @root begin # perform all this only on rank 0, aka the "root" rank
-        if !isfile(filepath)
-            try 
-                Downloads.download(fileurl, filepath; progress=download_progress, verbose=true)
-            catch 
-                cmd = `wget --no-check-certificate -O $filepath $fileurl`
-                @root run(cmd)
-            end
-        end
+    fileurl  = url * "/" * filename # joinpath on windows creates the wrong url
+
+    @root if !isfile(filepath) # perform all this only on rank 0, aka the "root" rank
+        Downloads.download(fileurl, filepath; progress=download_progress)
     end
     
     dataset = Dataset(filepath)
```

Comment:
Don't you need a synchronization here? (I don't know if `@root` implies a sync)

If the file is not there, the other MPI processes will move to the last line of this snippet before the file is downloaded and fail.

Reply:
the synchronization is in the `@root` macro so all the processes wait for that instruction to complete.